test/test_util_pkl5.py

```diff
@@ -593,6 +593,67 @@ def testMessage(self):
         self.assertEqual(msg, self.MessageType(MPI.MESSAGE_NO_PROC))
         self.assertNotEqual(msg, MPI.MESSAGE_NULL)
 
+    def testBcastIntra(self, msglist=None, check=None):
+        comm = self.COMM
+        size = comm.Get_size()
+        for smess in (msglist or messages):
```

Comment:
I don't think `messages` gets tested, only `msglist`? Is this intended?

Reply:
No, it is tested. At some point the `unitest` infrastructure will call `testcase.testBcastIntra()` without arguments, and then the `messages` case will be tested.

Reply:
I misread something, and also I forgot it's part of `unittest`, so this function would be called with the default args indeed (since it has a prefix `test`).